src/main/java/uk/gov/hmcts/darts/common/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.darts.common.service.impl;
```

Comment:
I'm not sure if this should be in common package.
Common package is for common code that other modules use. I don't think anything in the common module uses code outwith the common module.
I think introducing this code here that runs code in Daily List etc, might introduce a "tangle":-
https://structure101.com/help/java/studio/Content/restructure101/tangles.html

Perhaps we should move it to its own module.

Reply:
So my initial thinking was the shared code would be in common and the actual tasks would be in their respective packages but it probably makes more sense to put them in own package